### PR TITLE
Join card component

### PR DESCRIPTION
**Pull request by achen409**
image

### CODE REVIEW

**Comment by SafeDuck** [comment]
https://tailwindcss.com/docs/box-shadow

Add a box shadow like how it is in the figma

**Comment by achen409** [comment]
image

File: src/components/about/Card.jsx [hunk 1]
```diff
@@ -1,7 +1,21 @@
 import React from "react";
+import { FaDiscord } from "react-icons/fa";
 
 const Card = () => {
-  return <div>Card</div>;
+  return (
+    <div className="bg-white flex flex-col items-center justify-top pt-4 w-48 h-1/4 rounded-lg border-1 shadow-lg">
+      <div className="space-y-1 flex flex-col items-center justify-center">
+        <FaDiscord className="text-7xl" />
```
Comment (SafeDuck) [inline src/components/about/Card.jsx:8]: make it hiss-purple

**Comment by achen409** [comment]
image

File: src/components/about/Card.jsx [hunk 2]
```diff
@@ -1,7 +1,21 @@
 import React from "react";
+import { FaDiscord } from "react-icons/fa";
 
 const Card = () => {
```
Comment (SafeDuck) [inline src/components/about/Card.jsx:4]: CSS is good. Now add props: {text, icon, link} 

example of how to use these props: https://github.com/acm-ucr/asme-website/blob/dev/src/components/home/JoinCard.jsx

File: src/components/about/Card.jsx [hunk 3]
```diff
@@ -1,7 +1,23 @@
+import Link from "next/link";
 import React from "react";
 
-const Card = () => {
-  return <div>Card</div>;
+const Card = ({ text, icon, link }) => {
+  return (
+    <div className="bg-white flex flex-col items-center justify-top pt-4 pb-12 w-48 h-1/4 rounded-lg border-1 shadow-lg">
```
Comment (SafeDuck) [inline src/components/about/Card.jsx:6]: remove justify-top, and h-1/4

**Review by SafeDuck — CHANGES_REQUESTED** [review]

**Comment by achen409** [comment]
image

**Review by SafeDuck — APPROVED** [review]

**Pull request merged**